app/webpack/observations/identify/components/observation_modal.jsx

```diff
@@ -671,23 +671,33 @@ class ObservationModal extends React.Component {
             <ul className="inat-tabs">
               {activeTabs.map( tabName => (
                 <li key={`obs-modal-tabs-${tabName}`} className={activeTab === tabName ? "active" : ""}>
-                  <button
-                    type="button"
-                    className="btn btn-nostyle"
-                    onClick={e => {
-                      e.preventDefault( );
-                      chooseTab( tabName, { observation } );
-                      return false;
-                    }}
-                  >
-                    {
+                  {activeTabs.length > 1 ? (
+                    <button
+                      type="button"
+                      className="btn btn-nostyle"
+                      onClick={e => {
+                        e.preventDefault( );
+                        chooseTab( tabName, { observation } );
+                        return false;
+                      }}
+                    >
+                      {
                       tabTitles[tabName]
                       || translateWithConsistentCase(
                         _.snakeCase( tabName ),
                         { case: "upper", defaultValue: tabName }
                       )
                     }
```

Comment:
I'm a little surprised eslint allows this, but generally you want opening and closing curly brackets to be at the same indentation level, or on the same line.